### ISSUE TITLE
[mysql] java.sql.SQLException: No suitable driver 

### ISSUE THREAD

**Issue by Jiabao-Sun** [ververica/flink-cdc-connectors#643]
**Describe the bug**
java.sql.SQLException: No suitable driver

```
2021-11-23 17:34:34,013 INFO  com.ververica.cdc.connectors.shaded.com.zaxxer.hikari.HikariDataSource [] - connection-pool-xxx.xxx.xxx:3306 - Starting...
2021-11-23 17:34:34,024 ERROR com.ververica.cdc.connectors.mysql.debezium.DebeziumUtils    [] - Failed to open MySQL connection
java.lang.RuntimeException: Failed to get driver instance for jdbcUrl=jdbc:mysql://xxx.xxx.xxx:3306/?useInformationSchema=true&nullCatalogMeansCurrent=false&useUnicode=true&characterEncoding=UTF-8&characterSetResults=UTF-8&zeroDateTimeBehavior=CONVERT_TO_NULL
	at com.ververica.cdc.connectors.shaded.com.zaxxer.hikari.util.DriverDataSource.<init>(DriverDataSource.java:114) ~[blob_p-a1a1796103e7ef9d8b6691d201a3426342cdac2c-8669d6803806968f8d3f1d156fcef5a0:2.1.0]
	at com.ververica.cdc.connectors.shaded.com.zaxxer.hikari.pool.PoolBase.initializeDataSource(PoolBase.java:331) ~[blob_p-a1a1796103e7ef9d8b6691d201a3426342cdac2c-8669d6803806968f8d3f1d156fcef5a0:2.1.0]
	at com.ververica.cdc.connectors.shaded.com.zaxxer.hikari.pool.PoolBase.<init>(PoolBase.java:114) ~[blob_p-a1a1796103e7ef9d8b6691d201a3426342cdac2c-8669d6803806968f8d3f1d156fcef5a0:2.1.0]
	at com.ververica.cdc.connectors.shaded.com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:108) ~[blob_p-a1a1796103e7ef9d8b6691d201a3426342cdac2c-8669d6803806968f8d3f1d156fcef5a0:2.1.0]
	at com.ververica.cdc.connectors.shaded.com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81) ~[blob_p-a1a1796103e7ef9d8b6691d201a3426342cdac2c-8669d6803806968f8d3f1d156fcef5a0:2.1.0]
	at com.ververica.cdc.connectors.mysql.source.connection.PooledDataSourceFactory.createPooledDataSource(PooledDataSourceFactory.java:54) ~[blob_p-a1a1796103e7ef9d8b6691d201a3426342cdac2c-8669d6803806968f8d3f1d156fcef5a0:2.1.0]
	at com.ververica.cdc.connectors.mysql.source.connection.JdbcConnectionPools.getOrCreateConnectionPool(JdbcConnectionPools.java:51) ~[blob_p-a1a1796103e7ef9d8b6691d201a3426342cdac2c-8669d6803806968f8d3f1d156fcef5a0:2.1.0]
	at com.ververica.cdc.connectors.mysql.source.connection.JdbcConnectionFactory.connect(JdbcConnectionFactory.java:53) ~[blob_p-a1a1796103e7ef9d8b6691d201a3426342cdac2c-8669d6803806968f8d3f1d156fcef5a0:2.1.0]
	at io.debezium.jdbc.JdbcConnection.connection(JdbcConnection.java:872) ~[blob_p-a1a1796103e7ef9d8b6691d201a3426342cdac2c-8669d6803806968f8d3f1d156fcef5a0:2.1.0]
	at io.debezium.jdbc.JdbcConnection.connection(JdbcConnection.java:867) ~[blob_p-a1a1796103e7ef9d8b6691d201a3426342cdac2c-8669d6803806968f8d3f1d156fcef5a0:2.1.0]
	at io.debezium.jdbc.JdbcConnection.connect(JdbcConnection.java:413) ~[blob_p-a1a1796103e7ef9d8b6691d201a3426342cdac2c-8669d6803806968f8d3f1d156fcef5a0:2.1.0]
	at com.ververica.cdc.connectors.mysql.debezium.DebeziumUtils.openJdbcConnection(DebeziumUtils.java:62) ~[blob_p-a1a1796103e7ef9d8b6691d201a3426342cdac2c-8669d6803806968f8d3f1d156fcef5a0:2.1.0]
	at com.ververica.cdc.connectors.mysql.MySqlValidator.validate(MySqlValidator.java:68) ~[blob_p-a1a1796103e7ef9d8b6691d201a3426342cdac2c-8669d6803806968f8d3f1d156fcef5a0:2.1.0]
	at com.ververica.cdc.connectors.mysql.source.MySqlSource.createEnumerator(MySqlSource.java:156) ~[blob_p-a1a1796103e7ef9d8b6691d201a3426342cdac2c-8669d6803806968f8d3f1d156fcef5a0:2.1.0]
	at org.apache.flink.runtime.source.coordinator.SourceCoordinator.start(SourceCoordinator.java:128) ~[flink-dist_2.11-1.13.3.jar:1.13.3]
	at org.apache.flink.runtime.operators.coordination.RecreateOnResetOperatorCoordinator$DeferrableCoordinator.applyCall(RecreateOnResetOperatorCoordinator.java:291) ~[flink-dist_2.11-1.13.3.jar:1.13.3]
	at org.apache.flink.runtime.operators.coordination.RecreateOnResetOperatorCoordinator.start(RecreateOnResetOperatorCoordinator.java:70) ~[flink-dist_2.11-1.13.3.jar:1.13.3]
	at org.apache.flink.runtime.operators.coordination.OperatorCoordinatorHolder.start(OperatorCoordinatorHolder.java:194) ~[flink-dist_2.11-1.13.3.jar:1.13.3]
	at org.apache.flink.runtime.scheduler.DefaultOperatorCoordinatorHandler.startAllOperatorCoordinators(DefaultOperatorCoordinatorHandler.java:85) ~[flink-dist_2.11-1.13.3.jar:1.13.3]
	at org.apache.flink.runtime.scheduler.SchedulerBase.startScheduling(SchedulerBase.java:592) ~[flink-dist_2.11-1.13.3.jar:1.13.3]
	at org.apache.flink.runtime.jobmaster.JobMaster.startScheduling(JobMaster.java:955) ~[flink-dist_2.11-1.13.3.jar:1.13.3]
	at org.apache.flink.runtime.jobmaster.JobMaster.startJobExecution(JobMaster.java:873) ~[flink-dist_2.11-1.13.3.jar:1.13.3]
	at org.apache.flink.runtime.jobmaster.JobMaster.onStart(JobMaster.java:383) ~[flink-dist_2.11-1.13.3.jar:1.13.3]
	at org.apache.flink.runtime.rpc.RpcEndpoint.internalCallOnStart(RpcEndpoint.java:181) ~[flink-dist_2.11-1.13.3.jar:1.13.3]
	at org.apache.flink.runtime.rpc.akka.AkkaRpcActor$StoppedState.start(AkkaRpcActor.java:605) ~[flink-dist_2.11-1.13.3.jar:1.13.3]
	at org.apache.flink.runtime.rpc.akka.AkkaRpcActor.handleControlMessage(AkkaRpcActor.java:180) ~[flink-dist_2.11-1.13.3.jar:1.13.3]
	at akka.japi.pf.UnitCaseStatement.apply(CaseStatements.scala:26) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at akka.japi.pf.UnitCaseStatement.apply(CaseStatements.scala:21) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at scala.PartialFunction$class.applyOrElse(PartialFunction.scala:123) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at akka.japi.pf.UnitCaseStatement.applyOrElse(CaseStatements.scala:21) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:170) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:171) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at akka.actor.Actor$class.aroundReceive(Actor.scala:517) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at akka.actor.AbstractActor.aroundReceive(AbstractActor.scala:225) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:592) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at akka.actor.ActorCell.invoke(ActorCell.scala:561) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:258) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at akka.dispatch.Mailbox.run(Mailbox.scala:225) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at akka.dispatch.Mailbox.exec(Mailbox.scala:235) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [flink-dist_2.11-1.13.3.jar:1.13.3]
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [flink-dist_2.11-1.13.3.jar:1.13.3]
Caused by: java.sql.SQLException: No suitable driver
	at java.sql.DriverManager.getDriver(DriverManager.java:315) ~[?:1.8.0_202]
	at com.ververica.cdc.connectors.shaded.com.zaxxer.hikari.util.DriverDataSource.<init>(DriverDataSource.java:106) ~[blob_p-a1a1796103e7ef9d8b6691d201a3426342cdac2c-8669d6803806968f8d3f1d156fcef5a0:2.1.0]
	... 42 more
```

**Environment :**
 - Flink version :  1.13.3
 - Flink CDC version:  2.1.0
 - Database and version: 8.0
 - Flink on Yarn

**Comment by zhuxiaoshang** [comment]
@Jiabao-Sun hello, how do you solve this problem, i also met it.

**Comment by Jiabao-Sun** [comment]
Closed #659
